[2025-09-02T21:16:24.133+0800] INFO - local_task_job_runner.py:123 - _execute() - ::group::Pre task execution logs
[2025-09-02T21:16:24.367+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_dummy_tasks_dag.end scheduled__2025-08-31T18:00:00+00:00 [queued]>
[2025-09-02T21:16:24.603+0800] INFO - taskinstance.py:2613 - are_dependencies_met() - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_dummy_tasks_dag.end scheduled__2025-08-31T18:00:00+00:00 [queued]>
[2025-09-02T21:16:24.615+0800] INFO - taskinstance.py:2866 - _check_and_change_state_before_execution() - Starting attempt 1 of 2
[2025-09-02T21:16:24.932+0800] INFO - taskinstance.py:2889 - _check_and_change_state_before_execution() - Executing <Task(BashOperator): end> on 2025-08-31 18:00:00+00:00
[2025-09-02T21:16:24.967+0800] INFO - standard_task_runner.py:72 - _start_by_fork() - Started process 331 to run task
[2025-09-02T21:16:25.033+0800] INFO - standard_task_runner.py:104 - _start_by_fork() - Running: ['***', 'tasks', 'run', 'example_dummy_tasks_dag', 'end', 'scheduled__2025-08-31T18:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/example_dummy_tasks_dag.py', '--cfg-path', '/tmp/tmpsefotz9k']
[2025-09-02T21:16:25.056+0800] INFO - standard_task_runner.py:105 - _start_by_fork() - Job 18: Subtask end
[2025-09-02T21:16:26.066+0800] INFO - task_command.py:467 - task_run() - Running <TaskInstance: example_dummy_tasks_dag.end scheduled__2025-08-31T18:00:00+00:00 [running]> on host a246a11defc8
[2025-09-02T21:16:28.110+0800] INFO - taskinstance.py:3132 - _execute_task_with_callbacks() - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-team' AIRFLOW_CTX_DAG_ID='example_dummy_tasks_dag' AIRFLOW_CTX_TASK_ID='end' AIRFLOW_CTX_EXECUTION_DATE='2025-08-31T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-31T18:00:00+00:00'
[2025-09-02T21:16:28.112+0800] INFO - taskinstance.py:731 - _execute_callable() - ::endgroup::
[2025-09-02T21:16:28.383+0800] INFO - subprocess.py:78 - run_command() - Tmp dir root location: /tmp
[2025-09-02T21:16:28.394+0800] INFO - subprocess.py:88 - run_command() - Running command: ['/usr/bin/bash', '-c', 'echo "Hello from Airflow! Success"']
[2025-09-02T21:16:28.494+0800] INFO - subprocess.py:99 - run_command() - Output:
[2025-09-02T21:16:28.497+0800] INFO - subprocess.py:106 - run_command() - Hello from Airflow! Success
[2025-09-02T21:16:28.499+0800] INFO - subprocess.py:110 - run_command() - Command exited with return code 0
[2025-09-02T21:16:28.883+0800] INFO - taskinstance.py:340 - _run_raw_task() - ::group::Post task execution logs
[2025-09-02T21:16:28.884+0800] INFO - taskinstance.py:352 - _run_raw_task() - Marking task as SUCCESS. dag_id=example_dummy_tasks_dag, task_id=end, run_id=scheduled__2025-08-31T18:00:00+00:00, execution_date=20250831T180000, start_date=20250902T131624, end_date=20250902T131628
[2025-09-02T21:16:29.138+0800] INFO - local_task_job_runner.py:266 - handle_task_exit() - Task exited with return code 0
[2025-09-02T21:16:29.524+0800] INFO - taskinstance.py:3895 - _schedule_downstream_tasks() - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T21:16:29.656+0800] INFO - local_task_job_runner.py:245 - _execute() - ::endgroup::
[2025-09-02T21:16:30.107+0800] INFO - standard_task_runner.py:217 - _read_task_utilization() - Process not found (most likely exited), stop collecting metrics
